distributed/scheduler.py

```diff
@@ -1963,6 +1963,7 @@ def identity(self, comm=None):
             "id": str(self.id),
             "address": self.address,
             "services": {key: v.port for (key, v) in self.services.items()},
+            "time_started": self.time_started,
```

Comment:
Maybe just `"started"` to make things shorter?  Definitely not something I care a ton about though.